packages/metals-languageclient/src/getJavaHome.ts

```diff
@@ -26,23 +27,26 @@ export type JavaVersion = "11" | "17" | "21";
  * @param javaVersion metals.javaVersion value as read from the configuration or default
  */
 export async function getJavaHome(
-  javaVersion: JavaVersion
+  javaVersion: JavaVersion,
+  outputChannel: OutputChannel
 ): Promise<string | undefined> {
-  const fromEnvValue = await fromEnv(javaVersion);
+  const fromEnvValue = await fromEnv(javaVersion, outputChannel);
```

Comment:
Any chance to forward coursier downloading information?

Reply:
We are forwarding courier downloading info.